frontend/components/AddItemForm.js

```diff
@@ -77,28 +96,34 @@ export default class AddItemForm extends React.Component {
       <ScrollView>
         <KeyboardAvoidingView behavior="position" enabled>
           <View style={styles.container}>
-          <Text h4>Add A Grocery Item</Text>
-          {INPUT_FIELDS.map((fieldProps) => {
-            return <Input
-                key={fieldProps.name}
-                onChangeText={(textInput) => this.setState({ [fieldProps.name]: textInput })}
-                value={this.state[fieldProps.name]}
-                containerStyle={styles.input}
-                {...fieldProps}
-                />
-          })}
-          <Button
-            title="  Scan UPC"
-            raised
-            icon={<Ionicons name="ios-camera" size={24} color={"white"}/>}
-            onPress={this.openScanner}
-            containerStyle={styles.input}
-            />
-          <Button
-            title="Submit Item"
-            raised
-            onPress={this.submitItemForm}
-            />
+            <Text h4 style={styles.input}>Add A Grocery Item</Text>
+            {this.state.image_uri !== "" && (<Image style={[styles.image, styles.input]} source={{uri: this.state.image_uri}}/>)}
+            <Button
+              title="Take A Picture"
+              raised
+              onPress={this.openCamera}
+              />
+            {INPUT_FIELDS.map((fieldProps) => {
+              return <Input
+                  key={fieldProps.name}
+                  onChangeText={(textInput) => this.setState({ [fieldProps.name]: textInput })}
+                  value={this.state[fieldProps.name]}
+                  containerStyle={styles.input}
+                  {...fieldProps}
+                  />
+            })}
+            <Button
```

Comment:
You could add a button style since all buttons seem to be raised and it would make it easier to modify the styles of buttons in the future if necessary.

Reply:
We currently just modify the container of a button so we have a containerStyle={styles.input} for now we don't need a separate style for buttons but will add one if the need arises